docs/en/stack/ml-biases.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[ml-biases]]
+=== Considering biases in {ml} processes
+
+{ml-cap} is meant to improve our everyday life in a wide palette of activities 
+across the globe. We use {ml} in science, healthcare, business, entertainment, 
+and many other fields. We make decisions based on {ml} results. It is, 
+therefore, crucial to keep in mind the possible biases that can corrupt results, 
+endanger fairness, and jeopardize the success of even the best-intended 
+initiative. Some of these biases can make their way into the {ml} results more 
+easily than we might think.
+
+Various possible biases can negatively influence the result of a {ml} analysis. 
+In this piece, we'll talk about statistical and social biases (without 
+addressing the numerous subtypes of them individually), and not about the 
+possible biases in the {ml} algorithms themselves. Statistical and social biases 
+can exist in the datasets used to train {ml} models or in the way the 
+predictions of these models are interpreted. These issues are part of a greater 
```

Comment:
Here, `These issues are part of a greater 
question about the ethics behind technology and there's no one solution for 
everyone.` , "there's no one for solution for everyone", because it is not specific enough to the topic under discussion (bias in ML) and strays a bit into a discussion of moral relativism (what x thinks is ok, is not what y thinks is ok, which is probably not what we want to allude to in the docs). Instead I'd rephrase like this

"These issues are part of a greater question about the ethics behind technology. While there are no quick technological fixes for removing bias from ML systems,  there are some basic practices you can use to detect and reduce these."